pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   logging: ^1.0.1
   meta: ^1.3.0
   test: ^1.17.3
+  yaml: ^3.1.1
```

Comment:
This can be in `dev_dependencies` since it's only used for testing

Reply:
sure!